carla_ros_bridge/src/carla_ros_bridge/ego_vehicle.py

```diff
@@ -127,6 +129,16 @@ def send_vehicle_msgs(self, frame, timestamp):
         vehicle_status.control.reverse = self.carla_actor.get_control().reverse
         vehicle_status.control.gear = self.carla_actor.get_control().gear
         vehicle_status.control.manual_gear_shift = self.carla_actor.get_control().manual_gear_shift
+        for wheel_idx in range(4):
+            wheel_location = VehicleWheelLocation(wheel_idx)
+            wheel_status = CarlaVehicleWheelStatus()
+            wheel_status.speed = self.carla_actor.get_wheel_speed(wheel_location) * 30.0 / math.pi
```

Comment:
why 30 here? Wheel radius?  linear speed = angular speed * pi * diameter, so angular speed should be linear speed / (diameter*pi) right?

Reply:
accident left these conversions in when debugging. speed will be deg/sec as commented in the msg definition